src/api/goods.ts

```diff
@@ -10,10 +10,14 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods
+      .sort((currentGood, nextGood) => (
+        currentGood.name.localeCompare(nextGood.name)
+      ))
+      .slice(0, 5));
 };
 
 export const getRedGoods = () => {
   return getAll()
-    .then(goods => goods); // get only red
+    .then(goods => goods.filter(good => good.color === 'red'));
```

Comment:
you need just color here
```suggestion
    .then(goods => goods.filter(({ color }) => = color === 'red'));
```